pkg/build/generated/clientset/typed/build/v1/buildconfig_expansion.go

```diff
@@ -0,0 +1,36 @@
+package v1
+
+import (
+	"errors"
+	"net/url"
+
+	build "github.com/openshift/origin/pkg/build/apis/build/v1"
+)
+
+type BuildConfigExpansion interface {
+	WebHookURL(name string, trigger *build.BuildTriggerPolicy) (*url.URL, error)
+}
+
+// ErrTriggerIsNotAWebHook is returned when a webhook URL is requested for a trigger
+// that is not a webhook type.
+var ErrTriggerIsNotAWebHook = errors.New("the specified trigger is not a webhook")
+
+// WebHookURL returns the URL for the provided build config name and trigger policy, or ErrTriggerIsNotAWebHook
+// if the trigger is not a webhook type.
+func (c *buildConfigs) WebHookURL(name string, trigger *build.BuildTriggerPolicy) (*url.URL, error) {
```

Comment:
I'd rather wire to a struct which contains a client-go/rest.Interface or whatever `RESTClient()` on the `BuildInterface` returns.

Reply:
yeah, i forgot I did this ;-) working on replacement